content/docs/atom-guide/dev-guide/shaders/azsl/shader-variant-options.md

```diff
@@ -1,10 +1,45 @@
 ---
 linktitle: Shader Variant Options
-title: Design of Shader Variant Options
+title: AZSL, Design Principles And Purpose Of Shader Variant Options
 description: Learn about the design principles and purpose of AZSL Shader Variant Options in the Atom Renderer. 
-toc: true
-weight: 300
-draft: true
```

Comment:
What does it mean to publish hidden? To me it's just another regular doc.

Reply:
This may have been set as "draft" previously, because it didn't have content. 
If it's `draft: true`, then it won't publish on the website. You are right to delete this line.